code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +402,238 @@ components:
       description: |
         Ports may be specified as a list of ranges or single ports.
     QosProfile:
+      description: |
+        Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          allOf:
+            - $ref: "#/components/schemas/QosProfileName"
+            - description: |
```

Comment:
In this case the description in `QosProfileName` is more complete, so I would remove this one.